lib/ImageGallery/RightChevron.js

```diff
@@ -0,0 +1,20 @@
+const RightChevron = ({ ...props }) => {
```

Comment:
Why destructure props instead of just accepting `(props)` as a simple argument?

Reply:
habit when I want one specific field and destructure the rest, not this case tho